src/server/controllers/account-transactions.js

```diff
@@ -293,7 +292,7 @@ const accountTransactions = async (req, res) => {
     variables.kind = variables.kind.split(',').map(toUpper).map(trim);
   }
 
-  variables.fetchHostFee = parseToBooleanDefaultFalse(variables.flattenHostFee);
+  variables.fetchHostFee = parseToBooleanDefaultFalse(req.query.flattenHostFee);
```

Comment:
`flattenHostFee` is not in variables anymore.

Reply:
Cool, I got why and I think it makes sense.